village/src/components/SmurfCard.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const SmurfCard = (props) => {
+  function deleteSmurf() {
+    props.deleteSmurf(props.id);
+  }
+
+  return (
+    <Card className="Smurf">
+      <XButton onClick={deleteSmurf}>X</XButton>
+      <Name onClick={() => props.history.push(`/smurfs/${props.id}`)}>
```

Comment:
👍 Nice. You might want to do something with the Smurf name so it's more obvious that it's clickable.